en/sim_gazebo_gz/gazebo-models.md

```diff
@@ -0,0 +1,73 @@
+# Gazebo Models Repository (PX4-gazebo-models)
+
+The [PX4-gazebo-models](https://github.com/PX4/PX4-gazebo-models) repository is used to store all [Gazebo](../sim_gazebo_gz/README.md) models and worlds that are supported by PX4.
```

Comment:
It would be good to add a sentence after this, once I understand it, that says something along the lines of: "PX4 automatically fetches this information when needed (or similar)".

I can't do that because not clear when it is fetched.

Reply:
Will add this.